rosette/rbl/rosette/configuration.rbl

```diff
@@ -44,5 +44,10 @@
 (load 'apropos		'silent)
 (load 'space 'silent)
 (load 'primitives 'silent)
-
-(load 'tests/README 'silent)
+(load 'tests/equiv 'silent)
+(load 'tests/reader-iso-8859-1 'silent)
+(load 'tests/reader-utf-8 'silent)
+(load 'tests/run-tests 'silent)
+(load 'tests/segfaults: 'silent)
```

Comment:
This will also make it sad. It's a directory.

Reply:
Fixed.